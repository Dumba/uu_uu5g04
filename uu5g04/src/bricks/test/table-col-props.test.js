/**
 * Copyright (C) 2019 Unicorn a.s.
 * 
 * This program is free software; you can use it under the terms of the UAF Open License v01 or
 * any later version. The text of the license is available in the file LICENSE or at www.unicorn.com.
 * 
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See LICENSE for more details.
 * 
 * You may contact Unicorn a.s. at address: V Kapslovne 2767/2, Praha 3, Czech Republic or
 * at the email: info@unicorn.com.
 */

import React from 'react';
import {shallow} from 'enzyme';
import UU5 from "uu5g04";
import "uu5g04-bricks";
import enzymeToJson from 'enzyme-to-json';
import TestTools from "../../core/test/test-tools.js";

const TagName = "UU5.Bricks.Table.Col";

const CONFIG = {
  mixins: [
    "UU5.Common.BaseMixin",
    "UU5.Common.ElementaryMixin",
    "UU5.Common.ColorSchemaMixin",
    "UU5.Common.NestingLevelMixin",
    "UU5.Common.PureRenderMixin"
  ],
  props: {
    span: {
      values:[1,2]
    },
  },
  requiredProps: {
    parent: shallow(
      <UU5.Bricks.Table.ColGroup id="parentIdColGroup" parent={
        shallow(<UU5.Bricks.Table id="parentId"/>).instance()
      } />
    ).instance()
  },
  opt: {
    shallowOpt: {
      disableLifecycleMethods: false
    },
    enzymeToJson: false
  }
};


describe(`${TagName}`, () => {
  TestTools.testProperties(TagName, CONFIG);
});


describe(`${TagName} docKit examples`, () => {

  it(`${TagName} should render without crash`, () => {

    const wrapper = shallow(
      <UU5.Bricks.Container id={"uuID01"}>
        <UU5.Bricks.Table id={"uuID"} header='Table'>
          {/*@@viewOn:0*/}
          <UU5.Bricks.Table.ColGroup id={"uuID02"}>
            <UU5.Bricks.Table.Col id={"uuID03"} span={1} colorSchema="blue"/>
            <UU5.Bricks.Table.Col id={"uuID04"} span={2} colorSchema="red"/>
          </UU5.Bricks.Table.ColGroup>
          {/*@@viewOff:0*/}
          <UU5.Bricks.Table.THead id={"uuID05"}>
            <UU5.Bricks.Table.Tr id={"uuID06"}>
              <UU5.Bricks.Table.Th id={"uuID07"} content='Name'/>
              <UU5.Bricks.Table.Th id={"uuID08"} content='Rank'/>
              <UU5.Bricks.Table.Th id={"uuID09"} content='Promotion prospects'/>
            </UU5.Bricks.Table.Tr>
          </UU5.Bricks.Table.THead>
          <UU5.Bricks.Table.TBody id={"uuID010"}>
            <UU5.Bricks.Table.Tr id={"uuID011"}>
              <UU5.Bricks.Table.Td id={"uuID012"} content='Rimmer'/>
              <UU5.Bricks.Table.Td id={"uuID013"} content='2nd class technician'/>
              <UU5.Bricks.Table.Td id={"uuID014"} content='comical'/>
            </UU5.Bricks.Table.Tr>
          </UU5.Bricks.Table.TBody>
        </UU5.Bricks.Table>
      </UU5.Bricks.Container>
    );
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  });
});










