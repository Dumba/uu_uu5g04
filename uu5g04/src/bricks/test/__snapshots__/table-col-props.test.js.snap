// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UU5.Bricks.Table.Col className 1`] = `
<col
  className="uu5-bricks-table-col my-classname"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 1`] = `
<col
  className="color-schema-default uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 2`] = `
<col
  className="color-schema-red uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 3`] = `
<col
  className="color-schema-red uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 4`] = `
<col
  className="color-schema-red-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 5`] = `
<col
  className="color-schema-red-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 6`] = `
<col
  className="color-schema-green uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 7`] = `
<col
  className="color-schema-green uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 8`] = `
<col
  className="color-schema-green-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 9`] = `
<col
  className="color-schema-green-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 10`] = `
<col
  className="color-schema-yellow uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 11`] = `
<col
  className="color-schema-yellow uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 12`] = `
<col
  className="color-schema-yellow-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 13`] = `
<col
  className="color-schema-yellow-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 14`] = `
<col
  className="color-schema-purple uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 15`] = `
<col
  className="color-schema-purple uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 16`] = `
<col
  className="color-schema-purple-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 17`] = `
<col
  className="color-schema-purple-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 18`] = `
<col
  className="color-schema-brown uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 19`] = `
<col
  className="color-schema-brown uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 20`] = `
<col
  className="color-schema-brown-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 21`] = `
<col
  className="color-schema-brown-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 22`] = `
<col
  className="color-schema-orange uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 23`] = `
<col
  className="color-schema-orange uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 24`] = `
<col
  className="color-schema-orange-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 25`] = `
<col
  className="color-schema-orange-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 26`] = `
<col
  className="color-schema-cyan uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 27`] = `
<col
  className="color-schema-cyan uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 28`] = `
<col
  className="color-schema-cyan-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 29`] = `
<col
  className="color-schema-cyan-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 30`] = `
<col
  className="color-schema-grey uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 31`] = `
<col
  className="color-schema-grey uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 32`] = `
<col
  className="color-schema-grey-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 33`] = `
<col
  className="color-schema-grey-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 34`] = `
<col
  className="color-schema-pink uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 35`] = `
<col
  className="color-schema-pink uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 36`] = `
<col
  className="color-schema-pink-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 37`] = `
<col
  className="color-schema-pink-rich uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col colorSchema 38`] = `
<col
  className="color-schema-pink uu5-bricks-table-col uu5-common-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col controlled 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col controlled 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col controlled 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col controlled 4`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col default props 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col disabled 1`] = `
<col
  className="uu5-bricks-table-col uu5-common-disabled"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col disabled 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col disabled 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col disabled 4`] = `
<col
  className="uu5-bricks-table-col uu5-common-disabled"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col docKit examples UU5.Bricks.Table.Col should render without crash 1`] = `
<div
  className="uu5-bricks-container uu5-bricks-container-spacing"
  id="uuID01"
  style={Object {}}
  title={null}
>
  <Table
    allowTags={Array []}
    bordered={false}
    className={null}
    colorSchema={null}
    condensed={false}
    content={null}
    controlled={true}
    disabled={false}
    dynamic={false}
    footer={null}
    header="Table"
    hidden={false}
    hover={false}
    id="uuID"
    ignoreInnerHTML={false}
    key="uuID"
    level={null}
    mainAttrs={null}
    name={null}
    nestingLevel={null}
    noIndex={false}
    pureRender={false}
    ref_={null}
    responsive={false}
    selected={false}
    striped={false}
    style={null}
    tooltip={null}
    underline={false}
  >
    <table-col-group
      allowTags={Array []}
      className={null}
      colorSchema={null}
      content={null}
      controlled={true}
      disabled={false}
      dynamic={false}
      hidden={false}
      id="uuID02"
      ignoreInnerHTML={false}
      mainAttrs={null}
      name={null}
      nestingLevel={null}
      noIndex={false}
      pureRender={false}
      ref_={null}
      selected={false}
      style={null}
      tooltip={null}
    >
      <TableCol
        className={null}
        colorSchema="blue"
        controlled={true}
        disabled={false}
        hidden={false}
        id="uuID03"
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        span={1}
        style={null}
        tooltip={null}
      />
      <TableCol
        className={null}
        colorSchema="red"
        controlled={true}
        disabled={false}
        hidden={false}
        id="uuID04"
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        span={2}
        style={null}
        tooltip={null}
      />
    </table-col-group>
    <table-thead
      allowTags={Array []}
      className={null}
      colorSchema={null}
      content={null}
      controlled={true}
      disabled={false}
      dynamic={false}
      hidden={false}
      id="uuID05"
      ignoreInnerHTML={false}
      mainAttrs={null}
      name={null}
      nestingLevel={null}
      noIndex={false}
      pureRender={false}
      ref_={null}
      selected={false}
      style={null}
      tooltip={null}
    >
      <table-tr
        allowTags={Array []}
        className={null}
        colorSchema={null}
        content={null}
        controlled={true}
        disabled={false}
        dynamic={false}
        hidden={false}
        id="uuID06"
        ignoreInnerHTML={false}
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        style={null}
        tooltip={null}
      >
        <table-th
          className={null}
          colSpan={null}
          colorSchema={null}
          content="Name"
          controlled={true}
          disabled={false}
          dynamic={false}
          hidden={false}
          id="uuID07"
          ignoreInnerHTML={false}
          mainAttrs={null}
          name={null}
          nestingLevel={null}
          noIndex={false}
          pureRender={false}
          ref_={null}
          rowSpan={null}
          selected={false}
          style={null}
          tooltip={null}
        />
        <table-th
          className={null}
          colSpan={null}
          colorSchema={null}
          content="Rank"
          controlled={true}
          disabled={false}
          dynamic={false}
          hidden={false}
          id="uuID08"
          ignoreInnerHTML={false}
          mainAttrs={null}
          name={null}
          nestingLevel={null}
          noIndex={false}
          pureRender={false}
          ref_={null}
          rowSpan={null}
          selected={false}
          style={null}
          tooltip={null}
        />
        <table-th
          className={null}
          colSpan={null}
          colorSchema={null}
          content="Promotion prospects"
          controlled={true}
          disabled={false}
          dynamic={false}
          hidden={false}
          id="uuID09"
          ignoreInnerHTML={false}
          mainAttrs={null}
          name={null}
          nestingLevel={null}
          noIndex={false}
          pureRender={false}
          ref_={null}
          rowSpan={null}
          selected={false}
          style={null}
          tooltip={null}
        />
      </table-tr>
    </table-thead>
    <table-tbody
      allowTags={Array []}
      className={null}
      colorSchema={null}
      content={null}
      controlled={true}
      disabled={false}
      dynamic={false}
      hidden={false}
      id="uuID010"
      ignoreInnerHTML={false}
      mainAttrs={null}
      name={null}
      nestingLevel={null}
      noIndex={false}
      pureRender={false}
      ref_={null}
      selected={false}
      style={null}
      tooltip={null}
    >
      <table-tr
        allowTags={Array []}
        className={null}
        colorSchema={null}
        content={null}
        controlled={true}
        disabled={false}
        dynamic={false}
        hidden={false}
        id="uuID011"
        ignoreInnerHTML={false}
        mainAttrs={null}
        name={null}
        nestingLevel={null}
        noIndex={false}
        pureRender={false}
        ref_={null}
        selected={false}
        style={null}
        tooltip={null}
      >
        <table-td
          className={null}
          colSpan={null}
          colorSchema={null}
          content="Rimmer"
          controlled={true}
          disabled={false}
          dynamic={false}
          hidden={false}
          id="uuID012"
          ignoreInnerHTML={false}
          mainAttrs={null}
          name={null}
          nestingLevel={null}
          noIndex={false}
          pureRender={false}
          ref_={null}
          rowSpan={null}
          selected={false}
          style={null}
          tooltip={null}
        />
        <table-td
          className={null}
          colSpan={null}
          colorSchema={null}
          content="2nd class technician"
          controlled={true}
          disabled={false}
          dynamic={false}
          hidden={false}
          id="uuID013"
          ignoreInnerHTML={false}
          mainAttrs={null}
          name={null}
          nestingLevel={null}
          noIndex={false}
          pureRender={false}
          ref_={null}
          rowSpan={null}
          selected={false}
          style={null}
          tooltip={null}
        />
        <table-td
          className={null}
          colSpan={null}
          colorSchema={null}
          content="comical"
          controlled={true}
          disabled={false}
          dynamic={false}
          hidden={false}
          id="uuID014"
          ignoreInnerHTML={false}
          mainAttrs={null}
          name={null}
          nestingLevel={null}
          noIndex={false}
          pureRender={false}
          ref_={null}
          rowSpan={null}
          selected={false}
          style={null}
          tooltip={null}
        />
      </table-tr>
    </table-tbody>
  </Table>
</div>
`;

exports[`UU5.Bricks.Table.Col hidden 1`] = `
<col
  className="uu5-bricks-table-col uu5-common-hidden"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col hidden 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col hidden 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col hidden 4`] = `
<col
  className="uu5-bricks-table-col uu5-common-hidden"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col id 1`] = `
<col
  className="uu5-bricks-table-col"
  id="myId"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col mainAttrs 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col name 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  name="myName"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 4`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 5`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 6`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 7`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 8`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 9`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 10`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 11`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 12`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 13`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 14`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 15`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 16`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 17`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 18`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col nestingLevel 19`] = `""`;

exports[`UU5.Bricks.Table.Col nestingLevel 20`] = `""`;

exports[`UU5.Bricks.Table.Col noIndex 1`] = `
<col
  className="uu5-bricks-table-col uu5-noindex"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col noIndex 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col noIndex 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col noIndex 4`] = `
<col
  className="uu5-bricks-table-col uu5-noindex"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col pureRender 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col pureRender 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col pureRender 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col pureRender 4`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col selected 1`] = `
<col
  className="uu5-bricks-table-col uu5-common-selected"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col selected 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col selected 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col selected 4`] = `
<col
  className="uu5-bricks-table-col uu5-common-selected"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col span 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  span={1}
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col span 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  span={2}
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col span 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  span={2}
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col span 4`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  span={1}
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col style 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col style 2`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col style 3`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col style 4`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Table.Col tooltip 1`] = `
<col
  className="uu5-bricks-table-col"
  id="uuID"
  style={Object {}}
  title="My Tooltip"
/>
`;
